<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="add_category_btn_layout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>431</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="items_list_layout"/>
   </widget>
   <widget class="QScrollArea" name="category_scroll_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>491</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea {
                background-color: white;
border: 1px white solid
            }
            
            QScrollBar:vertical {
                background: rgb(96,9,240); /* Цвет фона вертикального ползунка */
                width: 10px; /* Ширина вертикального ползунка */
            }

            QScrollBar::handle:vertical {
                background: rgb(96,9,240); /* Цвет вертикального ползунка */
            }

            QScrollBar::add-line:vertical {
                background: gray; /* Цвет фона кнопки прокрутки вниз */
            }

            QScrollBar::sub-line:vertical {
                background: gray; /* Цвет фона кнопки прокрутки вверх */
            }</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="category_scroll_content">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>491</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="category_load_btn">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>43</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
  width: 300px;
  height: 80px;
  border-radius: 5px;
  font-family: 'Lato', sans-serif;
  font-weight: 500;
  background: rgb(96,9,240);
  font-size: 20;</string>
    </property>
    <property name="text">
     <string>Обновить данные</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
